### DISCUSSION TITLE
out-of-source storage for test files

### DISCUSSION THREAD

**Discussion by nicolaslg** [LIHPC-Computational-Geometry/gmds#108] [Ideas]
Commit 96cc23469b646dd1238dafa7a102bc77977c1e26 adds some vtk mesh files in test_sample that are quite big.
I recommend looking for another way to store our test files, else we will end up with a bloated repo.

**Comment by benzaouille** [comment]
I will remove them from gmds and place them in an other repo, thank you

**Comment by franck-ledoux** [comment]
Just took a look how to do that. I will try to provide it in our organization with different repo. It implies to make our CI evolve.
